### PR TITLE
Backend JAX supports 1D diffusion equation in PINN forward

### PR DESCRIPTION
**Pull request by ZongrenZou**
Now both the forward and reverse modes support 1D diffusion equation in PINN forward.

### CODE REVIEW

File: examples/pinn_forward/diffusion_1d.py [hunk 1]
```diff
@@ -7,11 +7,17 @@
 # import torch
 # Backend paddle
 # import paddle
+# Backend jax
```
Comment (lululxvi) [inline examples/pinn_forward/diffusion_1d.py:10]: Move jax before paddle.

File: examples/pinn_forward/diffusion_1d.py [hunk 2]
```diff
@@ -33,6 +39,13 @@ def pde(x, y):
     #     + paddle.exp(-x[:, 1:])
     #     * (paddle.sin(np.pi * x[:, 0:1]) - np.pi ** 2 * paddle.sin(np.pi * x[:, 0:1]))
     # )
+    # Backend jax
```
Comment (lululxvi) [inline examples/pinn_forward/diffusion_1d.py:42]: Move jax before paddle.

Comment (ZongrenZou) [reply examples/pinn_forward/diffusion_1d.py]: Done.

File: examples/pinn_forward/diffusion_1d.py (hunk 1 above)
Comment (ZongrenZou) [reply examples/pinn_forward/diffusion_1d.py:8]: Done.

**Pull request merged**